bowler/tool.py

```diff
@@ -200,10 +200,21 @@ def refactor_dir(self, dir_name: str, *a, **k) -> None:
             filenames.sort()
             for name in filenames:
                 fullname = os.path.join(dirpath, name)
-                if not name.startswith(".") and self.filename_matcher(
-                    Filename(fullname)
-                ):
-                    self.queue_work(Filename(fullname))
+    # Skip hidden files (those starting with a dot)
+            
+max_retries = 3
+retries = 0
+
+while retries < max_retries:
```

Comment:
Something seems wrong with the indentation on this block, it displays at the left margin for me.